test/show.jl

```diff
@@ -248,6 +248,7 @@ end
 @test repr(:(∓ 1)) == ":(∓1)"
 @test repr(:(± 1)) == ":(±1)"
 
+is_juliarepr(x) = eval(Meta.parse(repr(x))) == x
```

Comment:
Maybe this is better?
```suggestion
roundtrip_repr(x) = eval(Meta.parse(repr(x)))
```
Then the `==` will be in the `@test` so the failure case will show both sides of the `==`

Reply:
Sure, I'll call it `eval_parse_repr(x)`